README.md

```diff
@@ -71,6 +71,11 @@ class HelloWorldHandler < HelloWorld
   # You can define your own initialize method to inject dependencies
 
   def method_name(request : TheRequest) : TheResponse
+
+    if request.text.nil? || request.text.size == 0
```

Comment:
This line won't compile since instance state can be changed between the two invocations of `request.text`, so I don't want to recommend it in documentation. Extracting a local variable or using a single method that performs both checks will work, though.

```suggestion
    unless request.text.presence
```